metagraph/src/graph/representation/succinct/dbg_succinct.cpp

```diff
@@ -648,14 +649,15 @@ bool DBGSuccinct::load_without_mask(const std::string &filename) {
     valid_edges_.reset();
 
     {
-        std::ifstream instream(utils::make_suffix(filename, kExtension), std::ios::binary);
+        std::unique_ptr<std::ifstream> in
+            = utils::open_ifstream(utils::make_suffix(filename, kExtension));
 
-        if (!boss_graph_->load(instream))
+        if (!boss_graph_->load(*in))
             return false;
 
-        mode_ = static_cast<Mode>(load_number(instream));
+        mode_ = static_cast<Mode>(load_number(*in));
 
-        if (!boss_graph_->load_suffix_ranges(instream))
+        if (!boss_graph_->load_suffix_ranges(*in))
             logger->warn("No index for node ranges could be loaded");
```

Comment:
Could you please double-check this? I'm noticing that this warning is being printed for all graphs

Reply:
This is a different warning...

Reply:
false alarm